solr/solr-ref-guide/src/major-changes-in-solr-9.adoc

```diff
@@ -170,6 +170,11 @@ Upgrade is to be done using Collection API MIGRATESTATEFORMAT action using a pre
 See for example https://solr.apache.org/guide/8_8/cluster-node-management.html#migratestateforma[Solr 8.8 Ref Guide].
 // Can't link directly to .adoc file, need to link to 8.something ref guide as MIGRATESTATEFORMAT no longer exists in 9.0.
 
+* If you're using a SolrJ CloudClient to connect to your Solr cluster, you must be using SolrJ version 8.10 or higher (8.x) when upgrading your SolrCloud from 8.x to 9.x.
+Otherwise, SolrJ will not be able to connect to the cluster once it has upgraded to Solr 9.
+Once you have upgraded all Solr clusters that the client is connecting to, you can upgrade the SolrJ client to 9.x.
+However until then, if you have a mix of Solr 9/Solr 8 clusters or nodes within a cluster, you must use SolrJ 8.10 or a higher minor version of 8.x.
```

Comment:
Can we combine this line with the line above :

```If you're using a SolrJ CloudClient to connect to your Solr cluster, you must be using SolrJ version 8.10 or higher (8.x) when upgrading your SolrCloud from 8.x to 9.x.```

I think they say the same thing already :)